pandas/tests/copy_view/test_methods.py

```diff
@@ -280,3 +280,21 @@ def test_head_tail(method, using_copy_on_write):
         # without CoW enabled, head and tail return views. Mutating df2 also mutates df.
         df2.iloc[0, 0] = 1
     tm.assert_frame_equal(df, df_orig)
+
+
+def test_assign(using_copy_on_write):
+    df = DataFrame({"a": [1, 2, 3]})
+    df_orig = df.copy()
+    df2 = df.assign()
+    df2._mgr._verify_integrity()
```

Comment:
```suggestion
```

For future tests, you can leave out this part (I don't think it is needed to repeat in every test)

Reply:
Noted, will do.